packages/@glimmer-workspace/integration-tests/test/trusted-html-test.ts

```diff
@@ -0,0 +1,49 @@
+import type { TrustedTypePolicy, TrustedTypesWindow } from 'trusted-types/lib';
+
+import { jitSuite, RenderTest, test } from '..';
+
+let policy: TrustedTypePolicy | undefined;
+if (typeof window !== 'undefined') {
+  let trustedTypes = (window as unknown as TrustedTypesWindow).trustedTypes;
+  if (trustedTypes?.createPolicy) {
+    policy = trustedTypes.createPolicy('test', {
+      createHTML: (s: string) => s,
+      createScript: (s: string) => s,
+      createScriptURL: (s: string) => s,
+    });
+  }
+}
+
+export class TrustedHTMLTests extends RenderTest {
+  static suiteName = 'TrustedHTML';
+
+  @test
+  'renders TrustedHTML similar to SafeString'() {
+    if (!policy) return;
+
+    let html = '<b>test\'"&quot;</b>';
+    this.registerHelper('trustedHTML', () => {
+      return policy?.createHTML(html);
+    });
+
+    this.render('<div>{{trustedHTML}}</div>');
+    this.assertHTML('<div><b>test\'""</b></div>');
+    this.assertStableRerender();
+  }
+
+  @test
+  'renders TrustedHTML in attribute context as string'() {
```

Comment:
this looks like it's (expectedly) still escaped (which we want!, because we don't want `trustedHTML` adding attributes -- should the test name be updated to explain this expectation?

Reply:
Theoretically this not ideal, trustedHTML potentially, can contain HTML entities that means developers will not notice that attributes double encoded. But I keep this similar to SafeString.
> because we don't want trustedHTML adding attributes
Yes, but I think ideally both SafeString and TrustedHTML should be HTML decoded in attribute value context and set as value raw (not escaping from attribute value context). 
> should the test name be updated to explain this expectation
Yes, I'll be more verbose in test.

Reply:
In the discord, in #dev-glimmer-vm, @chancancode suggested that this scenario should actually _error_, because TrustedHTML isn't meant for attributes, so the fact it can end up here is likely is a mistake at best, and we should help steer folks towards correct usage

Reply:
I agree with that statement, but I don't want to make that radical decision by myself in this PR. I know that it would create a ton of unnecessary work to fix every instance of sanitized values used in attribute context.